From d0a4c3c6f7dc4b627163a8af4a1b2145d5e19096 Mon Sep 17 00:00:00 2001
From: Timple Raj M <quic_timple@quicinc.com>
Date: Thu, 8 Feb 2024 12:46:35 +0530
Subject: [PATCH] soc: qcom: socinfo: print the CPU and SoC version

Add a print statement to display the CPU and SoC version

Change-Id: I96380f754ede5b74d5abc892adb88e33b5c34854
Signed-off-by: Timple Raj M <quic_timple@quicinc.com>
---
 drivers/soc/qcom/socinfo.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/soc/qcom/socinfo.c b/drivers/soc/qcom/socinfo.c
index f551f6be105c..ca2b62ed4c22 100644
--- a/drivers/soc/qcom/socinfo.c
+++ b/drivers/soc/qcom/socinfo.c
@@ -648,6 +648,9 @@ static int qcom_socinfo_probe(struct platform_device *pdev)
 	if (IS_ERR(qs->soc_dev))
 		return PTR_ERR(qs->soc_dev);
 
+	pr_info("CPU: %s, SoC Version: %s\n", qs->attr.machine,
+						qs->attr.revision);
+
 	pr_info("OEM_ID: %s, PROD_ID: %s\n", qs->attr.oem_id,
 						qs->attr.prod_id);
 
-- 
2.34.1

